website/source/docs/partnerships/index.html.md

```diff
@@ -0,0 +1,137 @@
+
+#### Platform:
+
+* [Consul-AWS for AWS Cloud Map](https://learn.hashicorp.com/consul/integrations/consul-aws)
+* [Consul Agent Cloud Auto-joining](https://www.consul.io/docs/agent/cloud-auto-join.html)
+
+#### Infrastructure:
+
+**Certificate Authority (CA)**
+
+* [Consul Certificate Management Documentation](https://www.consul.io/docs/connect/ca.html)
+* [Securing RPC Communication with TLS Encryption](https://learn.hashicorp.com/consul/security-networking/certificates)
+* [Consul Connect CA API](https://www.consul.io/api/connect/ca.html)
+
+**Identity and Access Management (IAM)**
+
+* [ACL Documentation and Guides](https://www.vaultproject.io/docs/configuration/storage/index.html)
+* [ACL API Documentation](https://www.consul.io/api/acl/acl.html)
+* [Securing Consul with ACLs](https://learn.hashicorp.com/consul/security-networking/production-acls)
+
+**Application Performance Monitoring (APM)**
+
+* [Consul Telemetry Documentation](https://www.consul.io/docs/agent/telemetry.html)
+* [Consul Cluster Monitoring and Metrics](https://learn.hashicorp.com/consul/day-2-operations/monitoring)
+* [Monitoring Consul with Telegraf](https://learn.hashicorp.com/consul/integrations/telegraf)
+
+**Logging**
+
+* [Consul Monitor Command Line](https://www.consul.io/docs/commands/monitor.html)
+* [Enable syslog via CLI](https://www.consul.io/docs/agent/options.html#enable_syslog)
+* [Enable syslog via config file](https://www.consul.io/docs/agent/options.html#_syslog)
+
+**Information Technology Service Management (ITSM)**
+
+* [Consul Service Registry](https://learn.hashicorp.com/consul/getting-started/servicesx)
+* [DNS Query Interface](https://learn.hashicorp.com/consul/getting-started/services#querying-services)
+* [HTTP API with Edge Triggers](https://learn.hashicorp.com/consul/getting-started/services#http-api)
+
+### 3. Develop and Test
+
+The only knowledge necessary to write a plugin is basic command-line skills and knowledge of the [Go programming language](http://www.golang.org). Use the plugin interface to develop your integration. All integrations should contain unit and acceptance testing.
+
+### 4. Review and Approval
+
+HashiCorp will review and approve your Consul integration. Please send an email to [consul-integration-dev@hashicorp.com](mailto:consul-integration-dev@hashicorp.com) with any relevant documentation, demos or other resources and let us know your integration is ready for review.
+
+### 5. Release
+
+At this stage, the Consul integration is fully developed, documented, reviewed and approved. Once released, HashiCorp will officially list the Consul integration.
+
+### 6. Support
+
+Many vendors view the release step to be the end of the journey, while at HashiCorp we view it to be the start. Getting the Consul integration built is just the first step in enabling users. Once this is done, ongoing effort is required to maintain the integration and address any issues in a timely manner.
```

Comment:
Besides stating support is available, can we add instructions on how partners can obtain support?